src/quantum/tp6.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import { frum } from '../utils/query_ops';
+import { TP6_PAGES } from './config';
+import DashboardPage from '../components/DashboardPage/index';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+
+const styles = {
+  body: {
+    backgroundColor: 'white',
+  },
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+  },
+};
+
+class TP6 extends React.Component {
+  render() {
+    const { classes, location } = this.props;
+    const params = new URLSearchParams(location.search);
+    const limits = { bits: params.get('bits') };
+
+    return (
+      <div className={classes.body}>
+        <DashboardPage
+          key={limits.bits}
```

Comment:
Yes, I guess it depends on if `limits.bits` is a prop, or `state`.  I imagined it as state (since I was pulling it from `location.search` at render time.  Since you suggest moving `bits` to the constructor, I see no need for the key either